hoomd/md/nlist.py

```diff
@@ -119,10 +124,12 @@ class Cell(_NList):
 
     Args:
         buffer (float):  Buffer width.
-        exclusions (tuple): ...
+        exclusions (tuple): Excludes pairs from the neighbor list, which 
```

Comment:
```suggestion
        exclusions (Tuple[str]): Excludes pairs from the neighbor list, which 
```
We are using explicit type annotations throughout the docs.